packages/shared/components/popups/RegisterProposalPopup.svelte

```diff
@@ -1,14 +1,16 @@
 <script lang="typescript">
     import { Button, TextInput, Text, TextType } from 'shared/components'
     import { localize } from '@core/i18n'
-    import { closePopup } from '@auxiliary/popup'
+    import { closePopup, openPopup } from '@auxiliary/popup'
     import { registerParticipationEvent } from '@core/profile-manager/api'
     import { isValidUrl } from '@core/utils'
     import { handleError } from '@core/error/handlers/handleError'
     import { showAppNotification } from '@auxiliary/notification'
+    import type { Auth } from '@iota/wallet'
+    import { activeProfile } from '@core/profile/stores'
 
     let eventId: string
-    let nodeUrl: string
+    let nodeUrl: string = $activeProfile?.clientOptions?.nodes[0]?.url
```

Comment:
I'm not sure this makes sense, since we plan to automatically register for proposals of nodes in the client options. We can leave it for now, since it speeds up development.